dist/index.html

```diff
@@ -1,17 +1,9 @@
 <!DOCTYPE html>
 <html>
 	<head>
-		<script src="js/bundle.js"></script>
 	</head>
 	<body>
-		<div id="hello-react"></div>
-		<div id="title">
-			Lab for Cedric
-		</div>
-		<div id="timeline-div">
-			<button id="update-timeline">Update Home Timeline</button>
-			<div id="error-div"></div>
-			<div id="tweets" class="hidden"></div>
-		</div>
+		<div id="root"></div>
+		<script src="js/bundle.js"></script>
```

Comment:
why was this moved?

Reply:
Are you talking about the script tag? I moved it to the bottom of the body to ensure the root element is rendered to the DOM before attempting to add any elements to it. Was the previous method better?